modules/calcite/src/main/java/org/apache/ignite/internal/processors/query/calcite/prepare/IgniteConvertletTable.java

```diff
@@ -53,6 +54,16 @@ private IgniteConvertletTable() {
         addAlias(IgniteOwnSqlOperatorTable.LENGTH, SqlStdOperatorTable.CHAR_LENGTH);
     }
 
+    /** {@inheritDoc} */
+    @Override public void registerOp(SqlOperator op, SqlRexConvertlet convertlet) {
+        super.registerOp(op, convertlet);
```

Comment:
If we want to extend a default operator, we should take in account overriding a default converter like `StandardConvertletTable#convertPlus(..)`.